### ISSUE TITLE
Installation Error Every Time

### ISSUE THREAD

**Issue by Alb11747** [OpenVPN/openvpn#375]
**Describe the bug**
Everytime I try to install OpenVPN I get this error "OpenVPNMSICA: tap_create_adapter: DiinstallDevice failed Error -536870397:"

![image](https://github.com/OpenVPN/openvpn/assets/40252880/f3461fef-ebc3-4c27-a189-f3003e3c9089)

**To Reproduce**
Download and run the [installer](https://openvpn.net/community-downloads/) (Though, it's machine specific).

**Expected behavior**
OpenVPN installs and run correctly.

**Version information (please complete the following information):**
 - OS: Microsoft Windows 11 Home - 10.0.22621 Build 22621
 - OpenVPN version: 2.6.5 to 2.5.9 (Tested on all the installers)

**Additional context**
It managed to get past this point even with the error but when I tried to connect it show an error that said window did not have a tap driver.

I had previously installed OpenVPN on this computer but when I got a notification that there was an update I downloaded and ran the new installer getting this error.

In Device Manager > Network Adapters, I see a "Unknown device" getting added everytime it gets to the error and removed when it rolls back the update.

**Comment by wolfgangihloff** [comment]
I see the same issue on Windows 11 Pro with 2.6.6 with latest updates as of today on new install of OpenVPN

![Screenshot 2023-09-28 174520](https://github.com/OpenVPN/openvpn/assets/162688/8d068854-c36e-480f-8d0f-24e8da9b36fb)

**Comment by chipitsine** [comment]
can you collect logs from "Event Viewer" --> "Windows Logs" --> "Application" --> (Source: MsiInstaller)

**Comment by chipitsine** [comment]
btw, do you have wireguard installed ?

**Comment by wolfgangihloff** [comment]
No, no Wireguard installed, I cannot upload in here the MsiInstaller events.evtx, but see screenshot
![Screenshot 2023-09-29 094523](https://github.com/OpenVPN/openvpn/assets/162688/265a7026-f86d-442b-bd87-0219fb8e32bf)
